File: packages/image_picker/android/src/test/java/io/flutter/plugins/imagepicker/FileUtilsTest.java
package io.flutter.plugins.imagepicker;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class FileUtilsTest extends FileUtils {

	@Test
	public void returnNullIfCannotGetCursor() {
		String nullString = FileUtils.getDataColumn(null, null, null,null);
		assertEquals(nullString, null);
	}

}

File: packages/image_picker/android/src/test/java/io/flutter/plugins/imagepicker/FileUtilsTest.kt
package io.flutter.plugins.imagepicker

import org.junit.Assert.assertNull
import org.junit.Test

class FileUtilsTest {

	@Test
	fun returnNullIfCannotGetCursor() {
		val nullString = FileUtils.getDataColumn(null, null, null,null)
		assertNull(nullString)
	}

}
